src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  @include flex(center, '');
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &.stars--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
Similar to line 9, ensure that the path '/src/images/star-active.svg' is correct and accessible from the CSS file's location.